test/unit/calculators/self_assessment_penalties_test.rb

```diff
@@ -408,6 +401,39 @@ def setup
           end
         end
 
+        context "HMRC Covid-19 Extension to 1 April for 2020-21" do
+          setup do
+            @calculator.tax_year = "2020-21"
+            @calculator.submission_method = "online"
+            @calculator.filing_date = Date.parse("2022-01-31")
+            @calculator.estimated_bill = SmartAnswer::Money.new(10_000)
+          end
+
+          context "the user has to pay interest on their debt if they pay between 31st Jan and 28th Feb - but no penalty" do
+            should "be no interest incurred prior to 31st Jan" do
+              @calculator.payment_date = Date.parse("2022-01-31")
+              assert_equal 0, @calculator.late_payment_penalty
+              assert_equal 0.0, @calculator.interest.to_f
+            end
+
+            should "incur interest payments between 31st Jan and 28th Feb" do
+              @calculator.payment_date = Date.parse("2022-02-01")
+              assert_equal 0, @calculator.late_payment_penalty
+              assert_equal 0, @calculator.interest.to_f
+
+              @calculator.payment_date = Date.parse("2022-02-28")
+              assert_equal 0, @calculator.late_payment_penalty
+              assert_equal 20.34, @calculator.interest.to_f
+            end
+          end
+
+          should "the user should have a late payment penalty as they are beyond the new deadline of 1st Mar" do
```

Comment:
Should this test state 1st April, rather than March? -> 'Late payment penalty (5% of outstanding bill): charged if payment after 1 April 2022.'

Reply:
Well spotted!